api/src/Entity/Activity.php

```diff
@@ -43,16 +44,21 @@
             security: 'is_granted("CAMP_MEMBER", object) or is_granted("CAMP_MANAGER", object)'
         ),
         new GetCollection(
-            normalizationContext: [
-                'groups' => [
-                    'read',
-                    'Activity:ActivityProgressLabel',
-                    'Activity:ActivityResponsibles',
-                    'Activity:ScheduleEntries',
-                ],
-            ],
+            normalizationContext: self::COLLECTION_NORMALIZATION_CONTEXT,
             security: 'is_authenticated()'
```

Comment:
Are you planning to remove this in a next step?

Reply:
As soon we know we dont use the Ressource anymore